README.md

```diff
@@ -101,97 +31,86 @@ Dependencies are updated with the follow nix command:
+1. Run `yarn nix:itn_selfnode` from `daedalus`.
+2. Run `yarn dev` from the subsequent `nix-shell`
+3. Once Daedalus has started, and has gotten past the loading screen, run `yarn itn:shelley:wallet:importer` from a new terminal window. This is only required if you wish to import some funded wallets. It is also possible to import funded legacy wallets by running `yarn itn:byron:wallet:importer` script.
 
-```bash
-$ export NETWORK=testnet
-$ yarn dev
-```
+#### ITN Rewards V1
 
-### Cardano Wallet API documentation
+1. Run `yarn nix:itn` from `daedalus`.
+2. Run `yarn dev` from the subsequent `nix-shell`
 
-While running Daedalus in development mode you can access Cardano Wallet API documentation on the following URL: https://localhost:8091/docs/v1/index/.
+#### QA Testnet
 
-# Testing
+1. Run `yarn nix:qa` from `daedalus`.
+2. Run `yarn dev` from the subsequent `nix-shell`
 
-You can find more details regarding tests setup within
-[Running Daedalus acceptance tests](https://github.com/input-output-hk/daedalus/blob/master/features/README.md) README file.
+#### Nightly Testnet
 
-**Notes:** Be aware that only a single Daedalus instance can run per state directory.
-So you have to exit any development instances before running tests!
+1. Run `yarn nix:nightly` from `daedalus`.
+2. Run `yarn dev` from the subsequent `nix-shell`
 
-## Wallet fault injection
+### Updating upstream dependencies (cardano-wallet, cardano-node & Jormungandr)
 
-General information about wallet fault injection can be found in the [Cardano's wallet-new README file](https://github.com/input-output-hk/cardano-sl/tree/develop/wallet-new#fault-injection).
+`Niv` is used to manage the version of upstream dependencies. The versions of these dependencies can be seen in `nix/sources.json`.
 
-`shell.nix` has support for passing the necessary flags:
+Dependencies are updated with the follow nix commands:
+- Update to the latest master: `nix-shell -A devops --run "niv update cardano-wallet"`
+- Update to a specific revision: `nix-shell -A devops --run "niv update cardano-wallet -a rev=1988f22895c45e12506ec83da0496ebdcdd17719"`
 
-- `--arg allowFaultInjection true` is necessary to enable any processing of faults, and
-- `--arg walletExtraArgs '[ "--somefault" ]'` can be used for enabling certain fault types at startup.
+#### Notes
 
-# Windows
+`nix-shell` also provides a script for updating `yarn.lock` file:
 
-This batch file requires [Node.js](https://nodejs.org/en/download/) and
-[7zip](https://www.7-zip.org/download.html).
+    nix-shell -A fixYarnLock
 
-    scripts/build-installer-win64.bat
+### Cardano Wallet Api documentation
 
-The result will can be found at `.\daedalus-*.exe`.
+Api documentation for edge `cardano-wallet` version: https://input-output-hk.github.io/cardano-wallet/api/edge/
 
-### CSS Modules
+### CSS modules
 
 This boilerplate out of the box is configured to use [css-modules](https://github.com/css-modules/css-modules).
```

Comment:
"This boilerplate"? 😅 after 3 years and thousands of hours of work on this project we should probably remove this part from the readme 😆

Reply:
😓 must have missed it somehow - removed now! https://github.com/input-output-hk/daedalus/pull/1953/commits/37e480fbe1648378806b1c982b99c813be1dfc3a